getting-started/troubleshooting.html.md

```diff
@@ -142,4 +142,29 @@ Use these two commands to see environment variables.
 ```
 flyctl secrets list
 flyctl config env
-```
\ No newline at end of file
+```
+
+## _Troubles with WireGuard_
```

Comment:
To get Wireguard connection working on OSx I had to add the DNS server manually (fdaa:0:2e91::3), using Network settings GUI. 

I also had to disable other non-wireguard VPNs